core/federated/RTI/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ target_include_directories(${RTI_LIB} PUBLIC ${IncludeDir}/modal_models)
 target_include_directories(${RTI_LIB} PUBLIC ${IncludeDir}/utils)
 
 if (NOT DEFINED LOG_LEVEL)
-    set(LOG_LEVEL 0)
+    set(LOG_LEVEL 2)
```

Comment:
This change has nothing to do with semantic versioning. But I don't believe we should disable all prints by default. Even the help string when calling `RTI --help` is dropped unless LOG_LEVEL is greater than or equal to 2.